### PR TITLE
fixed styling error follow flake8 for function and test file.

### PR DESCRIPTION
**Pull request by HazelJJJ**

### CODE REVIEW

**Review by trevorki — APPROVED** [review]

File: kmeaningful/show_clusters.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def show_clusters(data, clusters, centroids = None):
 
     centroids: 2d array
     Coordinates of cluster centroids
-
+cd
```
Comment (mikelynch416) [inline kmeaningful/show_clusters.py:22]: Was this supposed to get added?

**Comment by trevorki** [comment]
Checks that failed were other functions code not yet formatted

**Comment by mikelynch416** [comment]
@trevorki did you mean to close this without merging?

File: kmeaningful/show_clusters.py (hunk 1 above)
Comment (HazelJJJ) [reply kmeaningful/show_clusters.py]: Thanks Mike. Nice catch.

**Pull request merged**